msi/jenkins.wxs

```diff
@@ -43,28 +43,47 @@
     <Upgrade Id="${MSI_PRODUCTCODE}">
       <UpgradeVersion Minimum="0.0.0" Maximum="99.99.99" Property="PREVIOUSVERSIONINSTALLED" />
     </Upgrade>
+
+    <CustomAction Id="BackupJenkinsXmlFile" Script="vbscript">
+      <![CDATA[
+        jenkinsDirPath = session.Property("JENKINSDIR")
+        Set fsObj = CreateObject("Scripting.FileSystemObject")
+
+        If fsObj.FolderExists(jenkinsDirPath) Then
+            jenkinsXmlPath = jenkinsDirPath + "jenkins.xml"
+
+            If fsObj.FileExists(jenkinsXmlPath) Then
+                Set shellObj = CreateObject("WScript.Shell")
+                cmdToExecute = "cmd.exe /C copy """ + jenkinsXmlPath + """ """ + jenkinsXmlPath + ".backup"" /Y"
```

Comment:
🐛  As well for this, and for not including a numeric suffix -- not so much because I'm concerned with having multiple versions but because if something goes wrong in installation someone's instinct will be to abort and try again, which will override the old settings.